Fix/auto select (#252)
NordicSemiconductor/pc-nrfconnect-quickstart: Changelog.md

-   Support for the nRF54L15 DK.

> -   Added new learning resource links in the Learn step for the nRF9151 DK,
>     nRF9160 DK and nRF9161 DK.
> -   If only one device is connected during the Select step, it is automatically
>     selected.

Reviewer: ```suggestion
-   New learning resource links in the Learn step for the nRF9151 DK,
    nRF9160 DK and nRF9161 DK.
-   Automatic selection of the device if only one is connected when at the Select step.
```
